### DISCUSSION TITLE
API Github - list contributors

### DISCUSSION THREAD

**Discussion by abezard** [community/community#24898] [Apps, API and Webhooks]
<p>Hi,</p>
<p>Currently, when hitting the Github’s API to get the list of contributors for a specific repo:</p>
<pre><code>GET /repos/:owner/:repo/collaborators
</code></pre>
<p>The list of collaborators I get includes outside collaborators, organization members that are direct collaborators, organization members with access through team memberships, organization members with access through default organization permissions, and organization owners. ( as stated in the documentation <a href="https://developer.github.com/v3/repos/collaborators/#list-collaborators" rel="nofollow noopener">here</a> )</p>
<p>I would like to know if there’s a way to get, via the API, only the collaborators specified in the <strong>Settings &gt; Collaborators &amp; teams &gt; Collaborators</strong> box?</p>
<p>Thanks.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/abezard">@abezard</a>,</p>
<p>Thank you for being here! You can pass a parameter called <code>affiliation</code> with value <code>outside</code> to get the collaborators you’ve added that aren’t part of the organization the repo belongs to. Here is more info: <a href="https://developer.github.com/v3/repos/collaborators/#parameters" rel="nofollow noopener">https://developer.github.com/v3/repos/collaborators/#parameters</a>. I hope that helps!</p>
<p>Best,</p>
<p>Andrea</p>
